### PR TITLE
Create Schnpat_R13.5-test.md

### PR DESCRIPTION
**Pull request by if338**
### Description of Change

- Please provide reviews or sign-off

This fixes #22 

/cc: @githubteacher

### CODE REVIEW

File: Schnpat_R13.5-test.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Schnpat CEF R13.5 ToDos
+
+
+### Details:
+1. blablablabla 1
+2. blablablabla 2
+3. blablablabla 3
```
Comment (if338) [inline Schnpat_R13.5-test.md:7]: What about no. 4

Comment (if338) [reply Schnpat_R13.5-test.md:7]: And what about No. 5

File: Schnpat_R13.5-test.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Schnpat CEF R13.5 ToDos
+
+
+### Details:
```
Comment (gerald8858) [inline Schnpat_R13.5-test.md:4]: Too many blas

**Review by gerald8858 — APPROVED** [review]

**Comment by if338** [comment]
Thanks Gerald, this is fine :-)

**Review by if338 — COMMENTED** [review]
test

**Pull request merged**